### PR TITLE
Specify PSP's so its easier to check when upgrade what one should remove

### PR DESCRIPTION
**Pull request by FrankHarland**

### CODE REVIEW

File: docs/getting-started/create.md [hunk 1]
```diff
@@ -152,6 +152,7 @@ the ones your Magento backend doesn't support.
 
 - Remove `"@graphcommerce/[psp]"` from package.json
 - Remove all [psp] references from `pages/checkout/payment.tsx`
+- Currently these are: `[@graphcommerce/mollie-magento-payment, @graphcommerce/magento-payment-braintree]`
```
Comment (ErwinOtten) [inline docs/getting-started/create.md:155]: Is addressed mentioned at line 150

Comment (paales) [reply docs/getting-started/create.md:155]: Wasn't too clear for me when reading it first, having the full strings available is helpful here.

Comment (ErwinOtten) [reply docs/getting-started/create.md:155]: @FrankHarland suggested changes:

### Remove unused PSP's

The example has Payment Service Providers integrated. Remove
the ones your Magento backend doesn't support.

- Remove Payment Service integrations from package.json: `@graphcommerce/mollie-magento-payment` and/or `@graphcommerce/magento-payment-braintree`
- Remove Payment Service references from `pages/checkout/payment.tsx`

**Review by FrankHarland — COMMENTED** [review]
@ErwinOtten done!

**Pull request merged**